colossus/src/main/scala/colossus/protocols/http/HttpBody.scala

```diff
@@ -7,6 +7,8 @@ import scala.util.Try
 
 class HttpBody(private val body: Array[Byte]) {
 
+  protected[http] var contentType: String = ""
```

Comment:
Lets rename this variable to indicate what is being used for, maybe something like `suggestedContentTypeFromEncoder`